.github/workflows/emscripten.yml

```diff
@@ -71,4 +72,4 @@ jobs:
           pip install pytest matplotlib
           # Don't use the cache provider plugin, as it doesn't work with Pyodide
           # right now: https://github.com/pypa/cibuildwheel/issues/1966
-          # pytest -svra -p no:cacheprovider --pyargs skimage
+          $PYTEST -svra -p no:cacheprovider --pyargs skimage
```

Comment:
I uncommented this line in https://github.com/scikit-image/scikit-image/pull/7555/commits/c3e6af13338f33b49300568469bb0a2529be2535. Feel free to revert if I missed something here.